core/store/ocr_key_store.go

```diff
@@ -0,0 +1,145 @@
+package store
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+
+	"github.com/smartcontractkit/chainlink/core/store/models/ocrkey"
+)
+
+// OCRKeyStore holds the OCR keys in memory
```

Comment:
Why is it even necessary to do this? Just request from DB and decrypt once then pass straight into the OCR library... no?

Reply:
Gutted!